src/html/index.html

```diff
@@ -136,33 +136,37 @@ <h1 class="headline">About</h1>
 
         <section class="intro panel">
           <div class="panel__content">
-            <div class="about buddies">
+            <div class="about buddies" alt="our buddies">
               <h1 class="headline">Buddies</h1>
               <p class="intro-text">
                 <strong class="krig">Here</strong> is a selection of some of our awesome buddy companies that we work with and soem of our great clients that we <3
               </p>
               <div class="buddies-container">
-                <!-- <a href="https://oakwood.se/"><img class="buddy" src="assets/img/oakwood.png"></a> -->
```

Comment:
We might as well just remove all none-used buddies, so that they are not in the DOM on deploy. Not a huge issue, but could be done still! 
Also: The alt text here is useful, so keep those! :)